packages/clay-color-picker/src/Editor.tsx

```diff
@@ -242,7 +265,11 @@ export function Editor({
 									}
 								}}
 								type="text"
-								value={hex.toUpperCase().substring(0, 6)}
+								value={
+									hex.toUpperCase().slice(-2) === 'FF'
```

Comment:
Why are we doing this here? Is there any specific case?

Reply:
Its checking if hex color has no transparency (represented by the two last 'FF' digits). If we have, there is no need to display it. 

Since we want to add the 8 digit, is better to add the tow last digits?
Add `hex.toUpperCase()`

Reply:
Hmm I think this might be hardcoded check for `FF` the alpha in a hex could be other values too maybe something like this?

```jsx
hex.toUpperCase().substring(0, color.getAlpha() < 1 ? 6 : 8)
```

Reply:
Thanks @matuzalemsteles !